It tries to compile freeglut with fails with:

```
[1/2] "/opt/homebrew/Cellar/cmake/3.24.0/bin/cmake" -E chdir ".." "/opt/homebrew/Cellar/cmake/3.24.0/bin/cmake" "/Users/ec2-user/vcpkg/buildtrees/freeglut/src/v3.2.2-576480b0e3.clean" "-G" "Ninja" "-DCMAKE_BUILD_TYPE=Release" "-DCMAKE_INSTALL_PREFIX=/Users/ec2-user/vcpkg/packages/freeglut_arm64-ios" "-DFREEGLUT_BUILD_STATIC_LIBS=1" "-DFREEGLUT_BUILD_SHARED_LIBS=0" "-DFREEGLUT_BUILD_DEMOS=OFF" "-DINSTALL_PDB=OFF" "-DCMAKE_MAKE_PROGRAM=/opt/homebrew/bin/ninja" "-DCMAKE_SYSTEM_NAME=iOS" "-DBUILD_SHARED_LIBS=OFF" "-DVCPKG_CHAINLOAD_TOOLCHAIN_FILE=/Users/ec2-user/vcpkg/scripts/toolchains/ios.cmake" "-DVCPKG_TARGET_TRIPLET=arm64-ios" "-DVCPKG_SET_CHARSET_FLAG=ON" "-DVCPKG_PLATFORM_TOOLSET=external" "-DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_SYSTEM_PACKAGE_REGISTRY=ON" "-DCMAKE_INSTALL_SYSTEM_RUNTIME_LIBS_SKIP=TRUE" "-DCMAKE_VERBOSE_MAKEFILE=ON" "-DVCPKG_APPLOCAL_DEPS=OFF" "-DCMAKE_TOOLCHAIN_FILE=/Users/ec2-user/vcpkg/scripts/buildsystems/vcpkg.cmake" "-DCMAKE_ERROR_ON_ABSOLUTE_INSTALL_DESTINATION=ON" "-DVCPKG_CXX_FLAGS=" "-DVCPKG_CXX_FLAGS_RELEASE=" "-DVCPKG_CXX_FLAGS_DEBUG=" "-DVCPKG_C_FLAGS=" "-DVCPKG_C_FLAGS_RELEASE=" "-DVCPKG_C_FLAGS_DEBUG=" "-DVCPKG_CRT_LINKAGE=dynamic" "-DVCPKG_LINKER_FLAGS=" "-DVCPKG_LINKER_FLAGS_RELEASE=" "-DVCPKG_LINKER_FLAGS_DEBUG=" "-DVCPKG_TARGET_ARCHITECTURE=arm64" "-DCMAKE_INSTALL_LIBDIR:STRING=lib" "-DCMAKE_INSTALL_BINDIR:STRING=bin" "-D_VCPKG_ROOT_DIR=/Users/ec2-user/vcpkg" "-D_VCPKG_INSTALLED_DIR=/Users/ec2-user/vcpkg/installed" "-DVCPKG_MANIFEST_INSTALL=OFF"
FAILED: ../CMakeCache.txt 
"/opt/homebrew/Cellar/cmake/3.24.0/bin/cmake" -E chdir ".." "/opt/homebrew/Cellar/cmake/3.24.0/bin/cmake" "/Users/ec2-user/vcpkg/buildtrees/freeglut/src/v3.2.2-576480b0e3.clean" "-G" "Ninja" "-DCMAKE_BUILD_TYPE=Release" "-DCMAKE_INSTALL_PREFIX=/Users/ec2-user/vcpkg/packages/freeglut_arm64-ios" "-DFREEGLUT_BUILD_STATIC_LIBS=1" "-DFREEGLUT_BUILD_SHARED_LIBS=0" "-DFREEGLUT_BUILD_DEMOS=OFF" "-DINSTALL_PDB=OFF" "-DCMAKE_MAKE_PROGRAM=/opt/homebrew/bin/ninja" "-DCMAKE_SYSTEM_NAME=iOS" "-DBUILD_SHARED_LIBS=OFF" "-DVCPKG_CHAINLOAD_TOOLCHAIN_FILE=/Users/ec2-user/vcpkg/scripts/toolchains/ios.cmake" "-DVCPKG_TARGET_TRIPLET=arm64-ios" "-DVCPKG_SET_CHARSET_FLAG=ON" "-DVCPKG_PLATFORM_TOOLSET=external" "-DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_SYSTEM_PACKAGE_REGISTRY=ON" "-DCMAKE_INSTALL_SYSTEM_RUNTIME_LIBS_SKIP=TRUE" "-DCMAKE_VERBOSE_MAKEFILE=ON" "-DVCPKG_APPLOCAL_DEPS=OFF" "-DCMAKE_TOOLCHAIN_FILE=/Users/ec2-user/vcpkg/scripts/buildsystems/vcpkg.cmake" "-DCMAKE_ERROR_ON_ABSOLUTE_INSTALL_DESTINATION=ON" "-DVCPKG_CXX_FLAGS=" "-DVCPKG_CXX_FLAGS_RELEASE=" "-DVCPKG_CXX_FLAGS_DEBUG=" "-DVCPKG_C_FLAGS=" "-DVCPKG_C_FLAGS_RELEASE=" "-DVCPKG_C_FLAGS_DEBUG=" "-DVCPKG_CRT_LINKAGE=dynamic" "-DVCPKG_LINKER_FLAGS=" "-DVCPKG_LINKER_FLAGS_RELEASE=" "-DVCPKG_LINKER_FLAGS_DEBUG=" "-DVCPKG_TARGET_ARCHITECTURE=arm64" "-DCMAKE_INSTALL_LIBDIR:STRING=lib" "-DCMAKE_INSTALL_BINDIR:STRING=bin" "-D_VCPKG_ROOT_DIR=/Users/ec2-user/vcpkg" "-D_VCPKG_INSTALLED_DIR=/Users/ec2-user/vcpkg/installed" "-DVCPKG_MANIFEST_INSTALL=OFF"
-- The C compiler identification is AppleClang 13.1.6.13160021
-- The CXX compiler identification is AppleClang 13.1.6.13160021
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
CMake Error at /opt/homebrew/Cellar/cmake/3.24.0/share/cmake/Modules/FindPackageHandleStandardArgs.cmake:230 (message):
  Could NOT find X11 (missing: X11_X11_INCLUDE_PATH X11_X11_LIB)
Call Stack (most recent call first):
  /opt/homebrew/Cellar/cmake/3.24.0/share/cmake/Modules/FindPackageHandleStandardArgs.cmake:594 (_FPHSA_FAILURE_MESSAGE)
  /opt/homebrew/Cellar/cmake/3.24.0/share/cmake/Modules/FindX11.cmake:481 (find_package_handle_standard_args)
  /Users/ec2-user/vcpkg/scripts/buildsystems/vcpkg.cmake:826 (_find_package)
  CMakeLists.txt:329 (FIND_PACKAGE)


-- Configuring incomplete, errors occurred!
See also "/Users/ec2-user/vcpkg/buildtrees/freeglut/arm64-ios-rel/CMakeFiles/CMakeOutput.log".
See also "/Users/ec2-user/vcpkg/buildtrees/freeglut/arm64-ios-rel/CMakeFiles/CMakeError.log".
[2/2] "/opt/homebrew/Cellar/cmake/3.24.0/bin/cmake" -E chdir "../../arm64-ios-dbg" "/opt/homebrew/Cellar/cmake/3.24.0/bin/cmake" "/Users/ec2-user/vcpkg/buildtrees/freeglut/src/v3.2.2-576480b0e3.clean" "-G" "Ninja" "-DCMAKE_BUILD_TYPE=Debug" "-DCMAKE_INSTALL_PREFIX=/Users/ec2-user/vcpkg/packages/freeglut_arm64-ios/debug" "-DFREEGLUT_BUILD_STATIC_LIBS=1" "-DFREEGLUT_BUILD_SHARED_LIBS=0" "-DFREEGLUT_BUILD_DEMOS=OFF" "-DINSTALL_PDB=OFF" "-DCMAKE_MAKE_PROGRAM=/opt/homebrew/bin/ninja" "-DCMAKE_SYSTEM_NAME=iOS" "-DBUILD_SHARED_LIBS=OFF" "-DVCPKG_CHAINLOAD_TOOLCHAIN_FILE=/Users/ec2-user/vcpkg/scripts/toolchains/ios.cmake" "-DVCPKG_TARGET_TRIPLET=arm64-ios" "-DVCPKG_SET_CHARSET_FLAG=ON" "-DVCPKG_PLATFORM_TOOLSET=external" "-DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_SYSTEM_PACKAGE_REGISTRY=ON" "-DCMAKE_INSTALL_SYSTEM_RUNTIME_LIBS_SKIP=TRUE" "-DCMAKE_VERBOSE_MAKEFILE=ON" "-DVCPKG_APPLOCAL_DEPS=OFF" "-DCMAKE_TOOLCHAIN_FILE=/Users/ec2-user/vcpkg/scripts/buildsystems/vcpkg.cmake" "-DCMAKE_ERROR_ON_ABSOLUTE_INSTALL_DESTINATION=ON" "-DVCPKG_CXX_FLAGS=" "-DVCPKG_CXX_FLAGS_RELEASE=" "-DVCPKG_CXX_FLAGS_DEBUG=" "-DVCPKG_C_FLAGS=" "-DVCPKG_C_FLAGS_RELEASE=" "-DVCPKG_C_FLAGS_DEBUG=" "-DVCPKG_CRT_LINKAGE=dynamic" "-DVCPKG_LINKER_FLAGS=" "-DVCPKG_LINKER_FLAGS_RELEASE=" "-DVCPKG_LINKER_FLAGS_DEBUG=" "-DVCPKG_TARGET_ARCHITECTURE=arm64" "-DCMAKE_INSTALL_LIBDIR:STRING=lib" "-DCMAKE_INSTALL_BINDIR:STRING=bin" "-D_VCPKG_ROOT_DIR=/Users/ec2-user/vcpkg" "-D_VCPKG_INSTALLED_DIR=/Users/ec2-user/vcpkg/installed" "-DVCPKG_MANIFEST_INSTALL=OFF"
FAILED: ../../arm64-ios-dbg/CMakeCache.txt 
"/opt/homebrew/Cellar/cmake/3.24.0/bin/cmake" -E chdir "../../arm64-ios-dbg" "/opt/homebrew/Cellar/cmake/3.24.0/bin/cmake" "/Users/ec2-user/vcpkg/buildtrees/freeglut/src/v3.2.2-576480b0e3.clean" "-G" "Ninja" "-DCMAKE_BUILD_TYPE=Debug" "-DCMAKE_INSTALL_PREFIX=/Users/ec2-user/vcpkg/packages/freeglut_arm64-ios/debug" "-DFREEGLUT_BUILD_STATIC_LIBS=1" "-DFREEGLUT_BUILD_SHARED_LIBS=0" "-DFREEGLUT_BUILD_DEMOS=OFF" "-DINSTALL_PDB=OFF" "-DCMAKE_MAKE_PROGRAM=/opt/homebrew/bin/ninja" "-DCMAKE_SYSTEM_NAME=iOS" "-DBUILD_SHARED_LIBS=OFF" "-DVCPKG_CHAINLOAD_TOOLCHAIN_FILE=/Users/ec2-user/vcpkg/scripts/toolchains/ios.cmake" "-DVCPKG_TARGET_TRIPLET=arm64-ios" "-DVCPKG_SET_CHARSET_FLAG=ON" "-DVCPKG_PLATFORM_TOOLSET=external" "-DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_SYSTEM_PACKAGE_REGISTRY=ON" "-DCMAKE_INSTALL_SYSTEM_RUNTIME_LIBS_SKIP=TRUE" "-DCMAKE_VERBOSE_MAKEFILE=ON" "-DVCPKG_APPLOCAL_DEPS=OFF" "-DCMAKE_TOOLCHAIN_FILE=/Users/ec2-user/vcpkg/scripts/buildsystems/vcpkg.cmake" "-DCMAKE_ERROR_ON_ABSOLUTE_INSTALL_DESTINATION=ON" "-DVCPKG_CXX_FLAGS=" "-DVCPKG_CXX_FLAGS_RELEASE=" "-DVCPKG_CXX_FLAGS_DEBUG=" "-DVCPKG_C_FLAGS=" "-DVCPKG_C_FLAGS_RELEASE=" "-DVCPKG_C_FLAGS_DEBUG=" "-DVCPKG_CRT_LINKAGE=dynamic" "-DVCPKG_LINKER_FLAGS=" "-DVCPKG_LINKER_FLAGS_RELEASE=" "-DVCPKG_LINKER_FLAGS_DEBUG=" "-DVCPKG_TARGET_ARCHITECTURE=arm64" "-DCMAKE_INSTALL_LIBDIR:STRING=lib" "-DCMAKE_INSTALL_BINDIR:STRING=bin" "-D_VCPKG_ROOT_DIR=/Users/ec2-user/vcpkg" "-D_VCPKG_INSTALLED_DIR=/Users/ec2-user/vcpkg/installed" "-DVCPKG_MANIFEST_INSTALL=OFF"
-- The C compiler identification is AppleClang 13.1.6.13160021
-- The CXX compiler identification is AppleClang 13.1.6.13160021
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
CMake Error at /opt/homebrew/Cellar/cmake/3.24.0/share/cmake/Modules/FindPackageHandleStandardArgs.cmake:230 (message):
  Could NOT find X11 (missing: X11_X11_INCLUDE_PATH X11_X11_LIB)
Call Stack (most recent call first):
  /opt/homebrew/Cellar/cmake/3.24.0/share/cmake/Modules/FindPackageHandleStandardArgs.cmake:594 (_FPHSA_FAILURE_MESSAGE)
  /opt/homebrew/Cellar/cmake/3.24.0/share/cmake/Modules/FindX11.cmake:481 (find_package_handle_standard_args)
  /Users/ec2-user/vcpkg/scripts/buildsystems/vcpkg.cmake:826 (_find_package)
  CMakeLists.txt:329 (FIND_PACKAGE)


-- Configuring incomplete, errors occurred!
See also "/Users/ec2-user/vcpkg/buildtrees/freeglut/arm64-ios-dbg/CMakeFiles/CMakeOutput.log".
See also "/Users/ec2-user/vcpkg/buildtrees/freeglut/arm64-ios-dbg/CMakeFiles/CMakeError.log".
ninja: build stopped: subcommand failed.
```

--- a/ports/jasper/vcpkg.json
+++ b/ports/jasper/vcpkg.json
@@ -4,6 +4,7 @@
   "port-version": 6,
   "description": "Open source implementation of the JPEG-2000 Part-1 standard",
   "homepage": "https://github.com/mdadams/jasper",
+  "license": null,
   "dependencies": [
     "libjpeg-turbo",
     {

--- a/versions/j-/jasper.json
+++ b/versions/j-/jasper.json
@@ -1,7 +1,7 @@
 {
   "versions": [
     {
-      "git-tree": "b9fae17938a71e1c43534ee8f15d9e2204ca23a1",
+      "git-tree": "fb8dca022aa130726431991cc9c70cc3162ea657",
       "version": "2.0.33",
       "port-version": 6
     },

